routes/general.js

```diff
@@ -4,5 +4,8 @@ const router = express.Router()
 router.route('/').get((req,res) => {
     res.render('home')
 })
+router.route('/forum').get((req,res)=> {
```

Comment:
there seems to be a little misunderstanding. 
- a static website is plain html/css/js, therefore:
- only javascript allowed is javascript within <script></script> tags or, linked with the "src" attributes of the same tags
- also means no npm libraries (so no node, express, all of that) :\
- if you want to add a library in a plain static site, you can either use what's called a cdn (basically an online resource, like bootstrap) or you have to locally include the file that you download from their website or something.

using node and express, you can make a backend service that you will be able to querry using http from your frontend (static website) using the "fetch" method.

Reply:
We were just talking about how we are going to have a static project and a backend project in the same repo, this would be on the backend project.

Reply:
oh, ok. so the forum would be generated on the backend and the "welcome" site would remain static html? @ddowy